### ISSUE TITLE
[files] Create folder from selection

### ISSUE THREAD

**Issue by hurradieweltgehtunter** [owncloud/web#9181]
It would be nice to have an option "Create folder from selection" when one or more files are selected.

**Comment by JanAckermann** [comment]
Even if @kulmann gave his thumbs up, I am not 100% sure if we won't spam the context menu with simple materials on board like in particular this case: copy & paste

**Comment by kulmann** [comment]
My thumbs up has a more sophisticated vision to it: I imagine that we will be able to provide such features as extensions in the future. And for an extension each ownCloud Admin (and later on hopefully even each user) can decide which of them are available. I.e. if someone wants to have the option in the context menu, because they do this all day and it speeds up their processes, they can just use it.

Since we're not there, yet, I'm not convinced that we want that in the general product at the moment. But we can let this issue here sit in the issue tracker for a while until our extension system is ready for it (@dschmidt is working on that at the moment, so hopefully something still for 2023).

**Comment by hurradieweltgehtunter** [comment]
Totally agree on the timeline. One piece of thought though: I like the idea to have extensions but IMO the set of file/folder operations should be fixed – and of course – well-designed, concept and layout wise. It makes it far easier for development and the user is presented one, unified ownCloud context menu - **the** ownCloud context menu. I always like to use Apple with MacOs as an example: They don't f*** around. They have a clear, well-thought concept and that's it. Windows or even Linux is the master of customization, and we all know the potential struggle, which comes with that.

**Comment by lookacat** [comment]
My honest opinion is that users won't use it because they will rely on a workflow thats similar to what they already know. At least for me even if I knew the option exists I probably wouldn't use it.

**Comment by kulmann** [comment]
Ok guys, I put a `Status:Blocked` label on this to indicate that nobody should work on this at the moment. Needs extension system enhancements first.

**Comment by dj4oC** [comment]
Before this can be acted on, the following needs to be resolved:

- This issue carries a `Status:Blocked` label, explicitly set by a maintainer pending extension-system enhancements, with the reasoning that "create folder from selection" should ship as an optional extension rather than a core context-menu item once the extension system supports it.

Could someone confirm whether the extension system has since reached the point where this could ship as an extension, unblocking this issue?

🤖 Automated by Claude Code

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
